### PR TITLE
Support ```DataFrame.scale```. ```StandardScaler```

### PR DESCRIPTION
**Pull request by KathiaBarahona**
Supporting `DataFrame.scale`, this is based on sklearn's [scale](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html). Also added `DataFrame.std_pop` and `SeriesAbstractNumeric.std_pop`, since the final calculations from sklearn and using the sample standard deviation where not the same.

Also, it's possible to do smth like:

```python
from bach.preprocessing.scalers import StandardScaler

training_df = DataFrame(....)
testing_df = DataFrame(...)

# create new scaler + "fit" training_df
scaler = StandardScaler(training_df)

scaled_testing_df = scaler.transform(testing_df)
```

Decided not to implement `scaler.fit` because we actually don't fit the dataframe, we just create a query that calculates `mean` and `standard deviation` from the training dataset but nothing is performed in runtime.

@hendrik-obj Please let me know if this works for you

### CODE REVIEW

File: bach/bach/preprocessing/scalers.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
+"""
+Copyright 2021 Objectiv B.V.
```
Comment (KathiaBarahona) [inline bach/bach/preprocessing/scalers.py:2]:
```suggestion
Copyright 2022 Objectiv B.V.
```
oops

File: bach/bach/dataframe.py [hunk 2]
```diff
@@ -2395,6 +2395,20 @@ def std(self, axis=1, skipna=True, level=None, ddof: int = 1, numeric_only=False
         return self._aggregate_func('std', axis, level, numeric_only,
                                     skipna=skipna, ddof=ddof, **kwargs)
 
+    def std_pop(self, axis=1, skipna=True, level=None, ddof: int = 1, numeric_only=False, **kwargs):
```
[the remark this answers is not in the document]
Comment (KathiaBarahona) [reply bach/bach/dataframe.py:2398]: oh didn't know. will test it

[the remark this answers is not in the document]
Comment (KathiaBarahona) [reply bach/bach/dataframe.py:2398]: https://github.com/objectiv/objectiv-analytics/blob/faa2a425458defb44612172e806c57ddb251d153/bach/bach/series/series_numeric.py#L77-L79

:)

**Review by jansentom — APPROVED** [review]
Looks nice, thanks for the additions and the std-fix

**Pull request merged**